### PR TITLE
Added clip img icon as per OnTime #1350

### PR DESCRIPTION
**Pull request by jamesgardham**
Added clip img icon as per OnTime #1350

### CODE REVIEW

File: theme/theme.inc [hunk 1]
```diff
@@ -212,9 +213,10 @@ function islandora_newspaper_preprocess_islandora_newspaper_page_controls(array
   // JP2 Clip.
   if ($object['JP2']) {
     $jp2_ds = $object['JP2'];
-    $text = t('Clip Image');
+    $path = $base_url . '/' . drupal_get_path('module', 'islandora_newspaper');
+    $text = '<img src="' . $path . '/images/clip_icon.png" />';
```
Comment (jordandukart) [inline theme/theme.inc]: Use theme('image', $variables) here http://api.drupal.org/api/drupal/includes!theme.inc/function/theme_image/7? Also for future reference using $base_url is generally bad. Can use the url function or file_create_url.

File: theme/theme.inc [hunk 2]
```diff
@@ -212,10 +213,12 @@ function islandora_newspaper_preprocess_islandora_newspaper_page_controls(array
   // JP2 Clip.
   if ($object['JP2']) {
     $jp2_ds = $object['JP2'];
-    $text = t('Clip Image');
+    $module_path = drupal_get_path('module', 'islandora_newspaper');
+    $path = url("$base_url/$module_path/images/clip_icon.png", array('absolute' => TRUE, 'external' => TRUE));        
+    $image = theme_image(array('path' => $path, 'attributes' => array()));
```
Comment (adam-vessey) [inline theme/theme.inc]: Providing `$module_path/images/clip_icon.png` should suffice as the `path`...  No need to run it through url() on your own.

Comment (adam-vessey) [inline theme/theme.inc]: Also, the theme function should not be called directly... Should be called like: `theme('image', array([...]))`.

File: theme/theme.inc [hunk 3]
```diff
@@ -212,10 +213,12 @@ function islandora_newspaper_preprocess_islandora_newspaper_page_controls(array
   // JP2 Clip.
   if ($object['JP2']) {
     $jp2_ds = $object['JP2'];
-    $text = t('Clip Image');
+    $module_path = drupal_get_path('module', 'islandora_newspaper');
+    $path = url("$module_path/images/clip_icon.png", array('absolute' => TRUE, 'external' => TRUE));
```
Comment (jordandukart) [inline theme/theme.inc]: Shouldn't need to use the url function here as theme('image', ...) calls out to file_create_url (http://api.drupal.org/api/drupal/includes!theme.inc/function/theme_image/7).

File: theme/theme.inc [hunk 4]
```diff
@@ -212,10 +213,12 @@ function islandora_newspaper_preprocess_islandora_newspaper_page_controls(array
   // JP2 Clip.
   if ($object['JP2']) {
     $jp2_ds = $object['JP2'];
-    $text = t('Clip Image');
+    $module_path = drupal_get_path('module', 'islandora_newspaper');
+    $path = url("$module_path/images/clip_icon.png", array('absolute' => TRUE, 'external' => TRUE));        
+    $image = theme('image',array('path' => $path, 'attributes' => array()));
```
Comment (jordandukart) [inline theme/theme.inc]: Space between the comma and the array.

**Pull request merged**